frontend/__tests__/src/components/Metrics/ConfigStep/PipelineTool.test.tsx

```diff
@@ -123,11 +123,12 @@ describe('PipelineTool', () => {
 
   it('should show error message and error style when token is invalid', async () => {
     const { getByText } = setup()
+    const mockInfo = 'mockToken'
     const tokenInput = screen.getByTestId('pipelineToolTextField').querySelector('input') as HTMLInputElement
 
-    await userEvent.type(tokenInput, MOCK_INFO)
+    await userEvent.type(tokenInput, mockInfo)
```

Comment:
Why revert it

Reply:
Because I realize that it shouldn't be a constant which could be extracted to fixtures.ts file. mockInfo is what we type in token input box for unit tests, and it's value is defined by ourselves according to testing requirements. Although mockInfo have been used many times in both SourceControl and PipelineTool components, we still couldn't extract a constant to satisfy all tests. I'm sorry for making incorrect changes without careful thinking.